.evergreen/run-tests.sh

```diff
@@ -82,6 +82,10 @@ export MONGODB_MULTI_MONGOS_LB_URI="${MONGODB_MULTI_MONGOS_LB_URI}"
 
 # Run the tests, and store the results in a junit result file
 case "$TESTS" in
+   atlas*)
```

Comment:
Isn't `atlas*` going to conflict with `atlas-data-lake*` below? Should we drop the asterisks? According to https://stackoverflow.com/a/4555979/162228, they do function as a wildcard for the pattern.

Just realized this while reviewing https://github.com/mongodb/mongo-php-library/pull/1151.